Find404.scala

```diff
@@ -0,0 +1,61 @@
+import java.io._
+import scala.io.Source._
+import scala.io.StdIn._
+import scala.collection.mutable.ArrayBuffer
+
+var fileSet = ArrayBuffer[File]()
+var ignoreSet = ArrayBuffer[String]()
+val directorySet = new File(args(0))
+if(directorySet.exists() == false) {
+    println("This directory does not exist")
```

Comment:
I like to get into the habit of making this sort of thing more meaningful. Something like this goes a long way when you're trying to debug:

```
println(s"${directorySet.getName} does not exist")
```